.github/workflows/built-branch.yml

```diff
@@ -0,0 +1,65 @@
+name: Create a -built branch
+
+on:
+  workflow_call:
+    inputs:
+      php:
+        default: 8.0
+        required: false
+        type: number
+      node:
+        default: 16
+        required: false
+        type: number
+
+jobs:
+  built-branch:
+    name: Built Branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clear previous vendor
+        shell: bash
+        run: rm -rf vendor
+      - name: Install dependencies
+        uses: php-actions/composer@v6
```

Comment:
What are your thoughts about this GitHub action for `composer`: https://github.com/ramsey/composer-install? 

Maybe they are just doing the same thing differently. But I've been using this one and I'm curious how different this one might be.

It's not clear from their documentation, https://github.com/php-actions/composer#caching-dependencies-for-faster-builds, if this one caches by default.

Reply:
Good idea, I've used `ramsey/composer-install` before so I'll stick to that.